api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -32,7 +32,7 @@
 public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements OpenmrsMetadata {
 	
 	//***** Properties *****
-	@Column(name = "name", nullable = false, length = 255)
+	@Column(name = "name", length = 255)
```

Comment:
@alinmihaila99, did you have to make this change?

Reply:
In drug's mapping there was property(name) that can be null, but using annotations drug extended that class, and was trying to add something null in it in a test. Is other way to change this only for drug?

Reply:
@alinmihaila99, did you confirm within the `Drug.hbm.xml` the corresponding field is not nullable?
If so, you could use `@AttributeOverride` on `Drug.java` by redefining the field's `@Column` to  resolve this? 

See [documentation](https://docs.jboss.org/hibernate/jpa/2.1/api/javax/persistence/AttributeOverride.html#:~:text=Annotation%20Type%20AttributeOverride&text=May%20be%20applied%20to%20an,of%20one%20of%20its%20attributes).)

See here

Reply:
I did this and it worked! thanks @Ruhanga